wgsl/index.bs

```diff
@@ -1643,6 +1682,27 @@ Issue: (dneto): Do we have to explicitly list the type environment Gamma? That's
   <thead>
     <tr><td>Precondition<td>Conclusion<td>Notes
   </thead>
+  <tr>
+    <td>
+    <td>`mat2x2<f32>()` : mat2x2<f32><br>
+        `mat3x2<f32>()` : mat3x2<f32><br>
+        `mat4x2<f32>()` : mat4x2<f32>
+    <td>Null value (OpConstantNull)
+  <tr>
+  <tr>
+    <td>
+    <td>`mat2x3<f32>()` : mat2x3<f32><br>
+        `mat3x3<f32>()` : mat3x3<f32><br>
+        `mat4x3<f32>()` : mat4x3<f32>
+    <td>Null value (OpConstantNull)
+  <tr>
+  <tr>
+    <td>
+    <td>`mat2x4<f32>()` : mat2x4<f32><br>
+        `mat3x4<f32>()` : mat3x4<f32><br>
+        `mat4x4<f32>()` : mat4x4<f32>
+    <td>Null value (OpConstantNull)
+  <tr>
```

Comment:
Should we add `struct` and `array` in here as well for completeness?

Reply:
This table is only for matrix constructors.

https://github.com/gpuweb/gpuweb/pull/764/files#diff-ef124ac387cefcdf9d7c6a9e4965d189R1681

Reply:
Sorry, I mean add a table for structs and arrays similar to the ones we have for other types add show the null value converts to OpConstantNull

Reply:
Fixed in https://github.com/gpuweb/gpuweb/pull/764/commits/747e67798549ae2883d501c73b1ebb89135fa3cb